### PR TITLE
update openai.mdx : #5666

### PR DESCRIPTION
**Pull request by karthiknadar1204**
## Description

Please include a summary of the change and the issue it solves. 

**Fixes** #(issue)

## Type of change

(Please delete options that are not relevant)

- [ ] 🐛 Bug fix (non-breaking change which fixes an issue)
- [ ] ⚡ New feature (non-breaking change which adds functionality)
- [ ] 📢 Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] 📄 This change requires a documentation update

### What is the solution?

(Describe at a high level how the feature was implemented)

## Checklist:

- [ ] My code follows the style guidelines(PEP 8) of MindsDB.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have updated the documentation, or created issues to update them.
- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] I have shared a short loom video or screenshots demonstrating any new functionality.

### CODE REVIEW

**Comment by martyna-mindsdb** [comment]
Related to issue #5666

File: docs/custom-model/openai.mdx [hunk 1]
```diff
@@ -16,6 +16,11 @@ CREATE ML_ENGINE openai2
 FROM openai
 USING
   api_key = 'YOUR_OPENAI_API_KEY';
+  <Tip>
```
Comment (martyna-mindsdb) [inline docs/custom-model/openai.mdx:19]: Please take it out of the code block, so it is like this:

> FROM openai
> USING
>   api_key = 'YOUR_OPENAI_API_KEY';
> \```
>   \<Tip>
> \**MindsDB Pro**
> Please note that it is required to provide an OpenAI API key when using MindsDB Pro version.
> \</Tip>

**Comment by martyna-mindsdb** [comment]
@karthiknadar1204 
Please see the review comment above.

Also, please sign our CLA by leaving a comment here that contains this line: I have read the CLA Document and I hereby sign the CLA

**Comment by karthiknadar1204** [comment]
Read the review,working on it,will write the cla comment

**Comment by karthiknadar1204** [comment]
Screenshot 2023-04-24 at 10 15 21 PM
is this fine?

**Comment by karthiknadar1204** [comment]
@martyna-mindsdb

**Comment by martyna-mindsdb** [comment]
@karthiknadar1204 
Please move ``` from line 24 to line 19.

**Comment by karthiknadar1204** [comment]
@martyna-mindsdb done

**Pull request merged**